ProductService/ProductService/ProductService.API/Configuration/DataAccessLayerExtention.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.EntityFrameworkCore;
+using ProductService.DAL;
+using ProductService.DAL.Interfaces.Repositories;
+using ProductService.DAL.Repositories;
+
+namespace ProductService.API.Configuration;
+
+public static class DataAccessLayerExtention
+{
+    public static void ConfigureDataAccessLayerDependencies(this IServiceCollection services, IConfiguration configuration)
+    {
+        var connectionString = configuration["ConnectionStrings:DefaultConnection"];
```

Comment:
I would return `GetConnectionString` method for better code style
```suggestion
        var connectionString = configuration.GetConnectionString("DefaultConnection");
```